### ISSUE TITLE
Error starting proxy server: OSError(48, 'Address already in use')

### ISSUE THREAD

**Issue by ghost** [iwaxx/blog#10]
I always get an error saying "Error starting proxy server: OSError(48, 'Address already in use')
" when starting NA. Any ideas?

**Comment by debookee** [comment]
Yes lot of ideas !

Are you using last `v7.2.1` ?

This problem occurs when old processes haven't been stopped properly (typically process `mitmdump` in your case)
-> This problem should be fixed in 7.2.1 by checking on startup old process presence and proper kill.

As a workaround, don't kill only `mitmdump` process, there might be other process as well that weren't stop, you wouldn't solve entirely the issue. Best workaround is too reboot if you use previous version of Debookee.

To sum up : if you're using 7.2.1 and still have those errors after a proper reboot after upgrade, then contact us at support@iwaxx.com for further investigations.
